src/main/kotlin/com/cognifide/gradle/aem/internal/PropertyParser.kt

```diff
@@ -45,15 +45,21 @@ class PropertyParser(val project: Project) {
     }
 
     private fun prop(name: String): String? {
-        var value = project.properties[name] as String?
-        if (value == null) {
-            value = systemProps[name]
+        if (project.hasProperty(name)) {
```

Comment:
in configuration phase we cannot access all project properties, because unintentionally we will access Maven Publish plugin property which can be read only once.

https://github.com/researchgate/gradle-release/commit/4d31ad7095e10d4f55d43bb376bcc257be32de36

as discussed here: https://github.com/spring-gradle-plugins/dependency-management-plugin/issues/158